app/code/core/Mage/Cms/Helper/Page.php

```diff
@@ -109,7 +109,7 @@ protected function _renderPage(Mage_Core_Controller_Varien_Action  $action, $pag
 
         $contentHeadingBlock = $action->getLayout()->getBlock('page_content_heading');
         if ($contentHeadingBlock) {
-            $contentHeading = $this->escapeHtml($page->getContentHeading());
+            $contentHeading = $this->escapeHtml((string)$page->getContentHeading());
```

Comment:
```suggestion
            $contentHeading = $this->escapeHtml($page->getContentHeading());
```
Revert after https://github.com/OpenMage/magento-lts/pull/2884